client/lib/analytics/page-view-tracker/index.jsx

```diff
@@ -68,25 +68,39 @@ export class PageViewTracker extends React.Component {
 			path,
 			recorder = noop,
 			hasSelectedSiteLoaded,
+			isSelectedSiteNotRequested,
 			title,
-			properties,
 		} = this.props;
 
 		debug( `Queuing Page View: "${ title }" at "${ path }" with ${ delay }ms delay` );
 
-		if ( ! hasSelectedSiteLoaded || this.state.timer ) {
+		if ( this.state.timer ) {
+			return;
+		}
+
+		if ( isSelectedSiteNotRequested ) {
+			return this.recordViewWithProperties();
+		}
+
+		if ( ! hasSelectedSiteLoaded ) {
 			return;
 		}
 
 		if ( ! delay ) {
-			return recorder( path, title, 'default', properties );
+			return this.recordViewWithProperties();
 		}
 
 		this.setState( {
 			timer: setTimeout( () => recorder( path, title ), delay ),
```

Comment:
I'm not aware why `recorder` is called with different arguments here. Leaving it as is.

Reply:
which different arguments are you referring to?

Reply:
`recorder` is called with `path` and `title` in one case, and with the additional `'default'` and `properties` arguments in another case.

Reply:
it was added in #23911 and I'm not sure. could have been an oversight or it could have been related to `delay` - supposing that the properties might come after some initial fetch, and we shouldn't necessary track the page view until they are likely to be available.

this is me guessing.